XamarinCommunityToolkitSample/Xamarin.CommunityToolkit.Sample.csproj

```diff
@@ -21,10 +21,6 @@
     <EmbeddedResource Update="Pages\Effects\SafeAreaEffectPage.xaml">
```

Comment:
I guess this file shouldn't be affected at all

Reply:
I'm going to do a rebase... there are to many changes to the main project for a normal merge. Hold the review till later.